yarn-project/circuits.js/src/structs/public_call_stack_item.test.ts

```diff
@@ -1,28 +1,28 @@
-import { makePrivateCallStackItem } from '../tests/factories.js';
-import { PrivateCallStackItem } from './call_stack_item.js';
+import { makePublicCallStackItem } from '../tests/factories.js';
+import { PublicCallStackItem } from './public_call_stack_item.js';
 
-describe('PrivateCallStackItem', () => {
+describe('PublicCallStackItem', () => {
   it('serializes to buffer and deserializes it back', () => {
     const randomInt = Math.floor(Math.random() * 1000);
-    const expected = makePrivateCallStackItem(randomInt);
+    const expected = makePublicCallStackItem(randomInt);
     const buffer = expected.toBuffer();
-    const res = PrivateCallStackItem.fromBuffer(buffer);
+    const res = PublicCallStackItem.fromBuffer(buffer);
     expect(res).toEqual(expected);
   });
 
   it('serializes to field array and deserializes it back', () => {
     const randomInt = Math.floor(Math.random() * 1000);
-    const expected = makePrivateCallStackItem(randomInt);
+    const expected = makePublicCallStackItem(randomInt);
 
     const fieldArray = expected.toFields();
-    const res = PrivateCallStackItem.fromFields(fieldArray);
+    const res = PublicCallStackItem.fromFields(fieldArray);
     expect(res).toEqual(expected);
   });
 
   it('computes hash', () => {
     const seed = 9870243;
-    const PrivateCallStackItem = makePrivateCallStackItem(seed);
-    const hash = PrivateCallStackItem.hash();
+    const PublicCallStackItem = makePublicCallStackItem(seed);
+    const hash = PublicCallStackItem.hash();
```

Comment:
😬 yikes

Reply:
fixed in https://github.com/AztecProtocol/aztec-packages/pull/4381/commits/6729aa7767ccab408b639d10e082d546c12f87ef